src/devices/load.jl

```diff
@@ -0,0 +1,87 @@
+function get_component_to_pm(ix::Int, load::L) where {L <: PSY.StaticLoad}
```

Comment:
we shouldn't define the variable `L` for concrete types if it isn't used. It also makes the code hard to read. I mentioned this above.